examples/src/main/java/io/dapr/examples/secrets/SecretClient.java

```diff
@@ -34,7 +34,7 @@
   /**
    * Identifier in Dapr for the secret store.
    */
-  private static final String SECRET_STORE_NAME = "vault";
+  private static final String SECRET_STORE_NAME = "local-secret-store";
 
   /**
    * JSON Serializer to print output.
```

Comment:
I know this is outside the scope of this issue but can we change the example to just fetch keys from args and nothing hardcoded? It means that the example should be changed to pass both keys at once:

```sh
dapr run --components-path ./components/secrets --config ./src/main/java/io/dapr/examples/secrets/config.yaml --app-id secrets2 -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.secrets.SecretClient redisPassword random
```

I would also recommend keeping the try-catch block to retrieve the key since it has a nice logging for the error message.

Reply:
done